centrallix-doc/Report/chart.html

```diff
@@ -109,6 +109,16 @@ <h3 class="childproperties">Child Properties:</h3>
 <td class="description">The color to be used for the series.</td>
 </tr>
 <tr>
+<td class="name">legend_name</td>
```

Comment:
These .html files are generated from the report.xml data file - you'll want to edit the xml file and then re-generate the various other files using the Makefile.  Note that using the Makefile will overwrite these two .html files.

Reply:
The documentation should now reflect the latest functionality and is entered into the report.xml file.